d2l-sequence-viewer.js

```diff
@@ -275,9 +281,10 @@ class D2LSequenceViewer extends mixinBehaviors([
 	}
 	ready() {
 		super.ready();
-
-		const styles = JSON.parse(document.getElementsByTagName('html')[0].getAttribute('data-asv-css-vars'));
-		const navBarStyles = JSON.parse(document.getElementsByTagName('html')[0].getAttribute('data-css-vars'));
+		const styles =
+			this.dataAsvCssVars && JSON.parse(this.dataAsvCssVars) ||
+			JSON.parse(document.getElementsByTagName('html')[0].getAttribute('data-asv-css-vars'));
+		const navBarStyles =  JSON.parse(document.getElementsByTagName('html')[0].getAttribute('data-css-vars'));
```

Comment:
Nit: Weird spacing on the right of the equals
```suggestion
		const navBarStyles = JSON.parse(document.getElementsByTagName('html')[0].getAttribute('data-css-vars'));
```